lifetimes/fitters/pareto_nbd_fitter.py

```diff
@@ -147,9 +150,19 @@ def conditional_probability_alive(self, frequency, recency, T):
         x, t_x = frequency, recency
         r, alpha, s, beta = self._unload_params('r', 'alpha', 's', 'beta')
 
-        A_0 = np.exp(self._log_A_0([r, alpha, s, beta], x, t_x, T))
-        return 1. / (1. + (s / (r + s + x)) *
-                     (alpha + T) ** (r + x) * (beta + T) ** s * A_0)
+        A_0 = 0
+        if alpha >= beta:
```

Comment:
hm, on second look, they look identical (unless I am missing something), so I would remove the one inside the `else`.